src/quo/components/calendar/calendar/component_spec.cljs

```diff
@@ -41,7 +47,10 @@
         {:start-date start-date
          :end-date   end-date
          :on-change  on-change}])
-      (h/fire-event :press (h/query-by-text (str (time/day start-date))))
+      (h/fire-event :press
+                    (-> (str (time/day start-date))
+                        (h/query-all-by-text)
+                        first))
```

Comment:
may be better to extract this bit, cause it's repeated 5 times

Reply:
Yeah good call, I've added an extra helper function named `h/query-first-by-text` 👍